docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,399 @@ Risk factor mediation
+The only thing that remains is to find the RR of the direct effect:
+
+.. math::
+
+  RR_\text{r→o,direct} = \frac{RR_\text{r→o,total}}{\prod_{m \in M} (RR_\text{m→o})^{\Delta_\text{r→m}}}
+
+.. note::
+  We may calculate a direct RR below 1.
+  This is not logically incoherent (a risk factor could have a protective direct effect
+  that is counteracted by harmful indirect/mediated effects), but it may
+  also indicate problems in the GBD mediation estimation, particularly if a
+  protective effect doesn't make sense given domain knowledge about the
+  risk factor.
+
+  We'll deal with these situations on a case-by-case basis.
+
+Delta method
+^^^^^^^^^^^^
+
+If GBD used the delta method, there are fewer steps.
+We simply solve the GBD equation:
+
+.. math::
+
+  MF = \frac{(RR_\text{m→o})^{\Delta_\text{r→m}} - 1}{RR_\text{r→o,total} - 1}
+
+for :math:`\Delta_\text{r→m}`:
+
+.. math::
+
+  \Delta_\text{r→m} = \frac{MF * (RR_\text{r→o,total} - 1) + 1}{log(RR_\text{m→o})}
+
+Having arrived at :math:`\Delta_\text{r→m}`, the last steps are the same as
+in the previous section.
+
+Approach 2: consistent with GBD mediation, inconsistent with RR application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this approach, we use GBD's mediation factors as they were intended to be used,
+but this is not consistent with the multiplicative application of RRs.
+If we take this approach, we will **not** match population-level prevalence
+rates -- Approach 1 is preferred, but this is described for completeness.
+
+We still need to assume a linear effect of the risk on the mediator, and we still
+calculate this like so:
+
+.. math::
+
+  \Delta_\text{r→m} = \frac{MF * (RR_\text{r→o,total} - 1) + 1}{log(RR_\text{m→o})}
+
+As before, this can result in multiple different delta values for the same risk-mediator
+pair; we have to choose one or aggregate them somehow.
+
+With this approach, we calculate the RR of the direct effect using the equation
+
+.. math::
+
+  RR_\text{r→o,direct} = 1 + (RR_\text{r→o,total} - 1) * (1 - MF_m)
+
+when there is only one mediator :math:`m`, as described in the methods appendix, or
+
+.. math::
+
+  RR_\text{r→o,direct} = 1 + (RR_\text{r→o,total} - 1) * \prod_{m \in M}{(1 - MF_m)}
+
+when there are multiple mediators, which is used internally by GBD (e.g. when calculating PAFs).
+
+.. todo::
+  Find a good citation for GBD's methods here.
+  Currently the best we have is this code file: https://stash.ihme.washington.edu/projects/CCGMOD/repos/paf/browse/mediate_rr.R?until=2af3bdeabcfce96ea48a0d77e3e24e6edd2f287a&untilPath=mediate_rr.R
+
+.. note::
+  This multiplicative combination of multiple mediation factors is hard to interpret -- it seems
+  to imply that mediators *overlap* in what they mediate, which seems to contradict the idea of
+  a causal effect.
+
+Risk deletion (PAFs) in the presence of mediation
+-------------------------------------------------
```

Comment:
I think we should state more directly/highlight here that the direct effect of R->O should be used for PAF calculation rather than the total effect.